stratisd_cert.py

```diff
@@ -625,6 +625,17 @@ def test_pool_destroy_permissions(self):
 
         self._test_permissions(StratisDbus.pool_destroy, [pool_name], True)
 
+    def test_pool_set_fs_limit_too_low(self):
+        """
+        Test setting the pool filesystem limit too low fails.
+        """
+        pool_name = p_n()
+        pool_path = make_test_pool(pool_name, StratisCertify.DISKS[0:1])
+
+        self._unittest_command(
```

Comment:
The ```_unittest_command``` function doesn't work properly for the situation where there is a failure to set a property. I think it may be best to write a new function, something like ```_unittest_property_set``` which handles the specific case where the command involves setting a property.

Reply:
Through some experimentation, I see some hint at what to write.  

If I were to set the filesystem limit to something that would succeed, the `_unittest_command()` call fails with `TypeError: object of type 'NoneType' has no len()`.

If I set the filesystem limit to something guaranteed to fail (i.e.: 0), and surround the call with a try/except block to catch the DBusException:
```
--- a/stratisd_cert.py
+++ b/stratisd_cert.py
@@ -632,9 +632,12 @@ class StratisdCertify(StratisCertify):  # pylint: disable=too-many-public-method
         pool_name = p_n()
         pool_path = make_test_pool(pool_name, StratisCertify.DISKS[0:1])
 
-        self._unittest_command(
-            StratisDbus.pool_set_fs_limit(pool_path, dbus.UInt64(0)), dbus.UInt16(1)
-        )
+        try:
+            self._unittest_command(
+                StratisDbus.pool_set_fs_limit(pool_path, dbus.UInt64(0)), dbus.UInt16(1)
+            )
+        except dbus.exceptions.DBusException as err:
+            pass
 
     def test_filesystem_create(self):
         """
```

...the test passes.  So, for this `_unittest_property_set` function, I would need to provide whether the caller expects the "set property" operation to succeed or fail.

Reply:
Correct.

Reply:
I found another problem: the call to `StratisDbus.pool_set_fs_limit` is being made in the test, and it seems that any exception from it needs to be handled in that test, which seems cumbersome.  This almost seems like I should move that call into a helper function, perhaps into something in StratisDbus, that can be called by the `_unittest_set_param()` function.